packages/blade/src/components/Checkbox/CheckboxLabel.web.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import styled from 'styled-components';
+import type { CheckboxLabelProps } from './CheckboxLabel.d';
+
+const StyledCheckboxLabel = styled.label({
+  display: 'flex',
+  userSelect: 'none',
```

Comment:
Why are we disabling user selection here 🤔

Reply:
🤔 well TBH we can argue that this is not good for accessibility. But in other hand improves UX since now double clicking to check/uncheck wont cause the text to be selected. Not sure what should we do tho.

Reply:
Personally I feel we should keep the defaults. Double clicking for selection seems rare and ability to select text shouldn't be hampered unless there's a very good reason. Thoughts?

Reply:
Sure.